CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.15...3.26)
```

Comment:
I would make the minimum 3.16.3, because that's the minimum for ITK. ITK also specifies a maximum of 3.19 but I have very rarely had trouble building with newer versions